substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationParser.java

```diff
@@ -61,26 +60,14 @@ public void parseAndRegister(Path path) throws IOException {
         throw new JSONParserException(errorMessage);
     }
 
-    protected static void checkAttributes(Map<String, Object> map, String type, Collection<String> requiredAttrs, Collection<String> optionalAttrs) {
+    protected static void checkAttributes(Map<String, Object> map, String type, Collection<String> requiredAttrs) {
         List<String> unseenRequired = new ArrayList<>(requiredAttrs);
-        for (String key : map.keySet()) {
```

Comment:
Does that mean we're now accepting any field name in configurations without an error or warning?